portal-api/api-tests/v1/users/post-create-user.api-test.js

```diff
@@ -88,9 +88,9 @@ describe('a user', () => {
     describe('it creates the user', () => {
       it('it creates the user if all provided data is valid', async () => {
         await createUser(MOCK_USER);
-        const { status, body } = await as(aNonAdmin).get(BASE_URL);
+        const { status, body } = await as(ADMIN).get(BASE_URL);
 
-        expect(status).toEqual(200);
+        expect(status).toEqual(401);
```

Comment:
```suggestion
        const { status, body } = await as(anAdmin).get(BASE_URL);

        expect(status).toEqual(200);
```

Reply:
Apologies for not noticing this on our call - this should've been `anAdmin`, not `ADMIN`, as the latter is just a constant for the string `'admin'`!